include/lib/xlat_tables/xlat_tables_defs.h

```diff
@@ -73,7 +73,11 @@
 #define PAGE_SIZE_MASK		(PAGE_SIZE - U(1))
 #define IS_PAGE_ALIGNED(addr)	(((addr) & PAGE_SIZE_MASK) == U(0))
 
-#define XLAT_ENTRY_SIZE_SHIFT	U(3) /* Each MMU table entry is 8 bytes (1 << 3) */
+#if ARM_ARCH_MAJOR == 7 && !ARMV7_SUPPORTS_LARGE_PAGE_ADDRESSING
```

Comment:
The only remaining CI failure is a MISRA issue, you can fix it by changing this to:
```
#if (ARM_ARCH_MAJOR == 7) && !ARMV7_SUPPORTS_LARGE_PAGE_ADDRESSING
```

Reply:
Thanks, Done.